CMakeLists.txt

```diff
@@ -219,12 +220,6 @@ endif()
 # VFX 18 deprecations to transition to MINIMUM_* variables in OpenVDB 7.0.0
 
 set(FUTURE_MINIMUM_CMAKE_VERSION 3.12)
```

Comment:
Why not also bump this one too?

Reply:
I tried, but it immediately broke our CI because the default version of CMake being pulled in there is 3.9.

We can bump this, but we'll need to get the Docker containers upgraded so I figured this can be a separate effort. I'd like to consider whether we can get away with a lower CMake version as well, if are asking our users to install CMake from source, that's going to be a hurdle I'd rather not insist on unless it's absolutely necessary.

Question to @aloysbaillet and @michdolan - I notice that OpenColorIO uses CMake 3.11 as a minimum - how is this working through the ASWF Docker containers?

Reply:
Okay, fine by me to leave this out of this PR for now

Reply:
OpenColorIO is using the 2019 tag for all builds currently, which is CMake 3.12.

Reply:
@michdolan - ah that's good to know, thanks!